src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/components/CourseProductCourseRuns/EnrolledCourseRun.tsx

```diff
@@ -91,7 +94,7 @@ const EnrolledCourseRun = ({ enrollment }: Props) => {
           {isStarded && (
             <Button
               size="small"
-              href={enrollment.course_run.resource_link}
+              href={`/dashboard${generatePath(LearnerDashboardPaths.ORDER, { orderId })}`}
```

Comment:
You should only do that if the related course run is a product. Otherwise, we should continue to use the `resource_link` as link.